modules/caddytls/connpolicy.go

```diff
@@ -298,6 +311,13 @@ type ClientAuthentication struct {
 	// which are not in this list will be rejected.
 	TrustedLeafCerts []string `json:"trusted_leaf_certs,omitempty"`
 
+	// A list of client certificate validators, for additional
+	// verification. These can perform custom checks, like ensuring
+	// the certificate is not revoked.
+	ValidatorsRaw []json.RawMessage `json:"validators,omitempty" caddy:"namespace=tls.client_cert_validators inline_key=validator"`
```

Comment:
```suggestion
	ValidatorsRaw []json.RawMessage `json:"validators,omitempty" caddy:"namespace=tls.client_auth inline_key=validator"`
```

Just a bit shorter, still descriptive